ui/src/app/richskill/list/skills-list.component.ts

```diff
@@ -228,6 +232,12 @@ export class SkillsListComponent extends QuickLinksHelper {
         callback: (action: TableActionDefinition, skill?: ApiSkillSummary) => this.handleClickUnarchive(action, skill),
         visible: (skill?: ApiSkillSummary) => this.unarchiveVisible(skill)
       }),
+      new TableActionDefinition({
```

Comment:
The button is invisible because I understood Paola said the button should appears but is visible only when you search, the callback do nothing because doesn't work in `skill-list-component` but is necessary to pass the callback, the implementation is in `rich-skill-search-results ` because you only export when you do a search.